src/hb-buffer-serialize.cc

```diff
@@ -89,12 +89,13 @@ hb_buffer_serialize_format_from_string (const char *str, int len)
 const char *
 hb_buffer_serialize_format_to_string (hb_buffer_serialize_format_t format)
 {
-  switch ((unsigned) format)
+  switch (format)
```

Comment:
Keep the cast.  We should cast in all cases of using these kinds of enums in switch.  I think otherwise compiler is free to assume that value is only one of the ones declared in the enum.